packages/core/admin/admin/src/components/UnstableGuidedTour/Tours.tsx

```diff
@@ -0,0 +1,78 @@
+import type { State, Action } from './Context';
+
+/* -------------------------------------------------------------------------------------------------
+ * Tours
+ * -----------------------------------------------------------------------------------------------*/
+
+const tours = {
+  contentManager: createTour('contentManager', [
+    {
+      name: 'TEST',
+      content: () => (
+        <>
+          <div>This is TEST</div>
+        </>
+      ),
+    },
+  ]),
+} as const;
+
+type Tours = typeof tours;
+
+/* -------------------------------------------------------------------------------------------------
+ * Tour factory
+ * -----------------------------------------------------------------------------------------------*/
+
+type TourStep<P extends string> = {
+  name: P;
+  content: Content;
+};
+
+type Content = ({
```

Comment:
just for a sense of readability, isn't it better to declare the Content type before the TourStep type?

Reply:
I think things are going to get moved around a lot so personally wouldn't worry about it right now but if you prefer I move it I will just let me know